analytics/partner.js

```diff
@@ -8,4 +8,16 @@
   analyticsHooks.on('partner:non-claimed', function (options) {
     analytics.track('Non-claimed partner page', _.extend(options, {nonInteraction: 1}))
   })
+
+  $('#partner-contact .email-gallery').click(function () {
+    analytics.track('Clicked Contact Gallery Via Email', {
+      gallery_id: sd.CURRENT_PATH.split('/')[1] // gallery id
```

Comment:
No, not typically. In this case I don't really have access to the `id` other than from `CURRENT_PATH` or querying the DOM which seems just as weird.